packages/terra-date-input/src/DateInput.jsx

```diff
@@ -719,7 +719,7 @@ class DateInput extends React.Component {
         {...this.props.yearAttributes}
         {...numberAttributes}
         refCallback={(inputRef) => { this.yearRef = inputRef; }}
-        label={this.props.intl.formatMessage({ id: 'Terra.date.input.yearLabel' })}
+        label={this.props.isInvalid ? `${'Error message'}, ${this.props.intl.formatMessage({ id: 'Terra.date.input.yearLabel' })}` : `${this.props.intl.formatMessage({ id: 'Terra.date.input.yearLabel' })}`}
```

Comment:
Same as above, 'Error message' should probably be internationalized.

Reply:
The string "Error message" is not required so it has been removed.